src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_dps_commands.py

```diff
@@ -258,11 +259,6 @@ def test_dps_linked_hub_lifecycle(self, group_name, group_location):
                  .format(dps_name, group_name, connection_string))
         self.cmd('az iot dps linked-hub delete --dps-name {} -g {} --linked-hub {}'.format(dps_name, group_name, hub_name))
 
-        # Create linked-hub using only connection string in lower case
-        self.cmd('az iot dps linked-hub create --dps-name {} -g {} --connection-string {}'
-                 .format(dps_name, group_name, connection_string.lower()))
-        self.cmd('az iot dps linked-hub delete --dps-name {} -g {} --linked-hub {}'.format(dps_name, group_name, hub_name))
-
```

Comment:
Why delete these commands in test?

Reply:
These commands do not work - as the key in the connection string is case sensitive. I have checked with the service and this behavior is expected.